### ISSUE TITLE
Question: TLS Verification Disable

### ISSUE THREAD

**Issue by allebone** [ansible-semaphore/semaphore#2314]
### Question

Howdy! We're attempting to connect Our current Semaphore Implementation to our internal Keycloak Server. As my business uses a Private CA, not self-signed, but an internally managed CA...the certificate for OIDC is showing as invalid TLS, and preventing connection. 

Is there an Environment Variable for disabling downstream TLS checking? 

or 

Is there a standardized way to add Root Certs when the container is running as a non-privileged users in this case?

### Related to

Configuration

**Comment by OctoNezd** [comment]
Hi, I am having same issue. Did you manage to find a proper workaround? I will add my CA to semaphore docker image, but it seems a bit hacky.

**Comment by Romloader** [comment]
Yes, didn't find any other option than adding the CA to the container unfortunately.

Something like:
- mapping the CA file in the container under `/usr/local/share/ca-certificates/yourCA.crt`
- Adding a `post_start` hook for the container (in case of docker compose, as an example)
```YML
post_start:
   - command: 'apk add ca-certificates && update-ca-certificates'
     user: root
```
(the example uses `apk` because the semaphore image as the time of writing is based on Alpine Linux)

It would still be better to have a less cumbersome and more stable solution (a way for semaphore to import the file)
